geowave-test/src/main/java/mil/nga/giat/geowave/test/GeoWaveTestEnvironment.java

```diff
@@ -75,7 +77,8 @@
 			"./target/accumulo_temp"); // breaks on windows if temp directory
 										// isn't on same drive as project
 
-	protected static boolean DEFER_CLEANUP = false;
+	protected static final AtomicBoolean DEFER_CLEANUP = new AtomicBoolean(
```

Comment:
I'm happy with the added safety, but am confident its not necessary given the usage of the mutex around any access to this flag.